package.json

```diff
@@ -15,7 +15,8 @@
     "packages/bundle-source",
     "packages/import-bundle",
     "packages/eventual-send",
-    "packages/produce-promise",
+    "packages/producePromise",
```

Comment:
I think this was a typo, because this PR did not also rename `packages/produce-promise/everything` to `packages/producePromise/`. This doesn't seem to be hurting anything (maybe `yarn` just ignores a workspace entry when the corresponding directory is missing?), but it effectively deletes `@agoric/produce-promise`. So @erights you might as well go ahead and make a PR to delete the `packages/produce-promise/` directory, this entry in `workspaces`, and the `.github/workflows/test-all-packages.yml` job.